### PR TITLE
Reverts beaker capacity back to original values

### PR DESCRIPTION
**Pull request by sneksnek**
## About The Pull Request

Reverts the normal, large, and x-large beakers back to their original capacities of 50u, 100u, and 120u respectively.

## Why It's Good For The Game

Beaker capacity was increased in the olden days so they had a wider range of factors and thus were more cleanly compatible with some formulas. This has some implications, which I believe generates more negative than positives.

Positives:
- You don't need to print an X-L beaker for a versatile 120u beaker.
- I can't think of anything else and that positive is debatably a negative.

Negatives:
- X-L beakers are mostly unused now since, from observation, most elect to use the 120u large beakers then make the jump to bluespace beakers.
- Medbay is less incentivized to produce plastic. Medbay would typically begin the round by producing some plastic to supply the lathe for printing X-L beakers. Since X-L beakers are now somewhat obsolete, many neglect to produce plastic unless requested. Roundstart, the only two important items that need plastic are X-L beakers and surgical drapes. Doctors spawn with surgical drapes and a bedsheet can easily substitute for it.
- Tiders or anyone waltzing into the pharmacy have immediate access to versatile beakers, enabling them to produce chemicals more freely than if they had proper beakers behind additional barriers.
- Chemical grenades are bigger without research. With 150u X-L beakers, you can have up to 300u in a grenade. This increases the size of your smoke or foam payloads. This also enables some thresholds on explosive chemicals. For example, water+potassium cannot reach a devastating 1 explosion with just 240u. It can with 300u. Nitrous oxide and gunpowder cannot reach devastating 2 explosions with just 240u. They can with 300u. In the most extreme example, stable nitroglycerin cannot reach devastating 4 explosions with just 240u. It can with 300u. With original values, you only have access to 240u grenades without research or potentially exploration. This devalues research a little.
- Chemmasters and their ilk have 120u capacities instead of 100u. This is comfortable enough that I've seen them go un-upgraded 99% of the time. With original values, you're incentivized to upgrade them.

There may be concerns for this mucking up recipes. However, I believe the vast majority of medbay players stuck to the 120u beakers until bluespace beakers were unlocked. With this change, all they need to do is produce plastic and print X-L beakers to use their same recipes. X-L beakers do not require research and can always be printed by producing plastic and deconstructing a few glass tables for glass if necessary.

## Proof Of Testing

I changed values in a single file and it compiled and all that. Here's a screeny:

beakers

## Changelog

:cl:
balance: Reverted the normal, large, and x-large beakers back to their original capacities of 50u, 100u, and 120u respectively.
/:cl:

### CODE REVIEW

**Comment by Swiftfeather** [comment]
1) If people want to squeeze more out of a Grenade they can. There's plenty of unused stuff in this game.
2) You just ask or order some.
3) Beaker size ain't gonna stop tiders making what they want, and this isn't really an issue here.
4) We don't allow mass bombing and mass casualty stuff, this isn't an issue for us. 
5) if they don't upgrade it who cares?

**Comment by sneksnek** [comment]
> 1. If people want to squeeze more out of a Grenade they can. There's plenty of unused stuff in this game.

Chemical grenades only accept bottles and beakers. They're limited to X-L beakers without research or exploration.

> 2. You just ask or order some.

It's healthier if they're incentivized to produce some for their own needs, then the rest goes to the crew.

> 3. Beaker size ain't gonna stop tiders making what they want, and this isn't really an issue here.

Yes it is. 50u and 100u is harder to work with and discourages creating chemicals yourself without the proper beakers. People do walk into the pharmacy here and I have to ask them to leave or toss them out.

> 4. We don't allow mass bombing and mass casualty stuff, this isn't an issue for us.

None of what I said immediately equates to murderboning or station-wide destruction. Foam, smoke, and explosions can be used in a controlled manner. They're all very effective tools for combat. Gibbing is permitted provided that the brain is retrievable.

> 5. if they don't upgrade it who cares?

You'll have to elaborate on this for me because this doesn't make sense. Upgrading things is supposed to be a part of the gameplay in ss13.

**Comment by lessthnthree** [comment]
I'm partial towards reverting it to TG because I don't see a compelling reason why it was ever adjusted from defaults to begin with. If there are compelling reasons for it to be 60, it should be able to be argued upstream.

**Comment by Swiftfeather** [comment]
I would have preferred a `git revert 2b890a6f5b5e7639543e3a46397a49af9bf55f6f` but I think shakers are fine being 120.


> I'm partial towards reverting it to TG because I don't see a compelling reason why it was ever adjusted from defaults to begin with. If there are compelling reasons for it to be 60, it should be able to be argued upstream.

https://github.com/Skyrat-SS13/Skyrat-tg/pull/3226

**Review by Swiftfeather — CHANGES_REQUESTED** [review]
I'll need the ci tests fixed too

/datum/unit_test/reagent_container_defaults
  Error: Reagent container /obj/item/reagent_containers/cup/beaker/oldstation: initial value of amount_per_transfer_from_this value (60) not found in possible_transfer_amounts list
  	FAILURE #1: Reagent container /obj/item/reagent_containers/cup/beaker/oldstation: initial value of amount_per_transfer_from_this value (60) not found in possible_transfer_amounts list at code/modules/unit_tests/reagent_container_defaults.dm:12
Error: FAIL /datum/unit_test/reagent_container_defaults 0.4s

**Comment by Swiftfeather** [comment]
Spoke with a bunch of people who med main but don't watch the github, seems fine.

**Comment by sippykot** [comment]
I found about this change when it was merged. I don't like it at all, and i don't think it was brought up on discord at all. It's a banket nerf to not only medical/sci but service, the round start machines etc. The arguments for it are poor, there are many sources of plastic and not like the station is plagued by plastic shortages - meanwhile larger beakers are medsci only. I don't think anyone asked for it, and seeing the reaction on discord it was not known by people playing med. The tider argument is bogus, nerfing grenades when the current balance is so heavily in favour of mass guncargo is not a good thing. Nerfing roundstart chem masters and reagent grinders is just straight up bad.

tl;dr big fat nerf that affects like half of the station with no good arguments in favour of it

_if you push it without public awareness, why was it not test-merged to get feedback?_

**Comment by sneksnek** [comment]
> I found about this change when it was merged. I don't like it at all, and i don't think it was brought up on discord at all.

There was no attempt to obfuscate this change. I understand some may not look at the github but Swift did say they spoke to others about it.

>It's a banket nerf to not only medical/sci but service, the round start machines etc. The arguments for it are poor, there are many sources of plastic and not like the station is plagued by plastic shortages - meanwhile larger beakers are medsci only. 

Nerfs are not intrinsically a bad thing. I believe my arguments are fairly objective. There were more reasons for the change than to simply encourage medbay to produce plastic. Service doesn't have any need for 120u large beakers; it only provides a little bit of ease and convenience.

>I don't think anyone asked for it

People don't need to ask for something for it to be done. There was an issue with the design so I put up a PR to try to fix it.

>The tider argument is bogus, 

No it's not. People should not be encouraged to tide into chemistry by giving them immediate access to versatile 120u beakers. As well, it's a general principle that anyone without access to a department has to use worse tools to fulfill a purpose. This is to encourage the people in the role to complete the task. 120u large beakers betray this principle.

>nerfing grenades when the current balance is so heavily in favour of mass guncargo is not a good thing. 

How so? Where was it decided that devastating 2 grenades and wider foam coverage were the counter to guncargo? The increased beaker size far predates current guncargo. If there is an issue with guncargo, it should be addressed at it's core and not be addressed with something as bizarre as increased chemical grenade size.

>Nerfing roundstart chem masters and reagent grinders is just straight up bad.

Again, how so? No reason is provided. Why did roundstart chemmasters and reagent grinders need increased space? What unique element of this downstream necessitates it? I cannot see why when we still use the base TG values for determining plant grinding/juicing yields. Chemmasters take very little effort to upgrade to 200u for mixing as well.

> tl;dr big fat nerf that affects like half of the station with no good arguments in favour of it

It only primarily affects medbay. I don't think anything in service is palpably affected by smaller beaker size since they're mostly used to store liquids or loosely mix chems they grow. They can upgrade their equivalent of the chemmaster to 200u for more than enough space to mix the odd thing. I did provide sufficient arguments.

> _if you push it without public awareness, why was it not test-merged to get feedback?_

The effects were not uncertain and I already addressed them in the PR.

**Comment by astoturf24** [comment]
This change was not ran by other medical/chemist mains nor was it posted about in the medical channel on discord.

The BENEFITS of 60/120u beakers (particularly the 60u) include it's divisibility into more numbers than a 50u beaker, making round start chemical mixing much more manageable and making them still usable and viable even late into the round

Addressing the perceived negatives, the over-arching point that's trying to be pushed across is incentivizing chemists to make plastic. I don't think this is the correct way to go about it. Chemists will just use the 50/100u beakers to make their chems instead of going out of their way to craft the 120u's just like the previous XL-Beakers they will go unused save by a few people who already make plastic at round start. This change will make no difference to this issue.

Medical is not the only department that needs plastic, other departments usually shout at chemists to make plastic as needed, and it is made. The same can be said for science and their research, or miners and their ores, this is just a fact of life on the station when one department is lacking. 

Tiders are going to tide no matter what size beakers are in chem and this is just padding

Grenades arent used offensively as much on bubberstation as on other station, it's a possibility yes but with the volumes your using examples of they would probably be so devastating that they would be in breach of rules and sicouraged.

Chemmaster upgrades are again, a player by player thing, some chemists care enough to upgrade them, most dont care and just use whatever is in the chem master. They will still not get upgraded, and chemists will use the 100u buffers in them. 

Above all, a huge number of us chem mains are against this change, enough to warrant multiple of us to come object to it on github, which alot of use dont watch like a hawk. We dont want this change, we were never aware of this change, and i feel like the majority of this community is against it.

**Comment by sneksnek** [comment]
> This change was not ran by other medical/chemist mains nor was it posted about in the medical channel on discord.

Swift said they brought it up with some people who main med.

> The BENEFITS of 60/120u beakers (particularly the 60u) include it's divisibility into more numbers than a 50u beaker, making round start chemical mixing much more manageable and making them still usable and viable even late into the round

You can still achieve this by printing XL beakers, which are easy to obtain and practically always available to anyone who would need it. You can print XL beakers without research.

> Addressing the perceived negatives, the over-arching point that's trying to be pushed across is incentivizing chemists to make plastic. I don't think this is the correct way to go about it. Chemists will just use the 50/100u beakers to make their chems instead of going out of their way to craft the 120u's just like the previous XL-Beakers they will go unused save by a few people who already make plastic at round start. This change will make no difference to this issue.

That's just a single point. I stressed none of the points as more important over the others. Chemists will use 120u beakers if they believe it is valuable enough to them. 120u confers a more significant benefit as per your own admission.

> Medical is not the only department that needs plastic, other departments usually shout at chemists to make plastic as needed, and it is made. The same can be said for science and their research, or miners and their ores, this is just a fact of life on the station when one department is lacking.
 
Other departments do need plastic and the most reliable, quick source is chemistry. This is why we should encourage them to produce it. There isn't a guarantee that anyone asked will act. Whether or not people do frequently do this (I will disagree with my own anecdote anyways by saying I'm fairly commonly asked to make it since it runs out) is irrelevant to the fact it's better design to encourage medbay to create their own before anyone runs into a scarcity of it and needs to depend on people who have no obligation to fulfill the request.

> Tiders are going to tide no matter what size beakers are in chem and this is just padding

Tiders will tide and try to use worse tools, correct. The purpose is to not completely eliminate the option but to discourage it. They shouldn't be encouraged to do so by providing them versatile beakers. Tiders or anyone not in the department are not supposed to have immediate access to good tools in medbay or any other department.

> Grenades arent used offensively as much on bubberstation as on other station, it's a possibility yes but with the volumes your using examples of they would probably be so devastating that they would be in breach of rules and sicouraged.

No it isn't. You can use bombs on bubberstation provided your destruction does not exceed that of a maxcap or syndicate bomb. It's written explicitly in the rules. Even 4 devastating 2 bombs wouldn't surpass this. Whether or not people use them often is irrelevant because they can and it can affect outcomes.

> Chemmaster upgrades are again, a player by player thing, some chemists care enough to upgrade them, most dont care and just use whatever is in the chem master. They will still not get upgraded, and chemists will use the 100u buffers in them.

Chemmaster upgrades are the easiest machines to upgrade, capacity-wise. You don't need special parts, you just need beakers above the defaults. Anyone with access to a chemmaster has access to large beakers roundstart to upgrade the machines to 200u, which is more than sufficient for most purposes. If a person does not care about their chemmaster capacity, then why would they suddenly start caring and be negatively affected now?

> Above all, a huge number of us chem mains are against this change, enough to warrant multiple of us to come object to it on github, which alot of use dont watch like a hawk. We dont want this change, we were never aware of this change, and i feel like the majority of this community is against it.

If I'm frank, I haven't seen many points addressing the change based on it's actual merit. Saying "me and the group of people I know don't like this" is not a strong enough argument on its own. I recommend trying it for a while, trying the new XL beakers, and avoiding having a knee-jerk reaction to a change. I proposed this change with the thought that it wouldn't drastically affect balance or flow since the option for a 120u beaker is still very accessible to the people that it matters for.

**Comment by sippykot** [comment]
The actual, overarching argument is "this will force chem to make plastic" which is not a problem that happens often and can be solved by asking, ordering a cheap crate from cargo, ordering a machine from cargo that can make infinite plastic from biomass... If you just want more plastic, could have added that recipe to the base biogenerator. Instead doing the wide-scale beaker nerf is not a good solution. Also thanks for confirming that service was just not considered at all - "the people that it matters for".

> "If I'm frank, I haven't seen many points addressing the change based on it's actual merit. Saying "me and the group of people I know don't like this" is not a strong enough argument on its own. I recommend trying it for a while, trying the new XL beakers, and avoiding having a knee-jerk reaction to a change. I proposed this change with the thought that it wouldn't drastically affect balance or flow since the option for a 120u beaker is still very accessible to the people that it matters for." 

...a change being disliked by the players IS an absolutely good reason for it to not be there. Telling people to "wait a while" on a PR that was fully merged is just trying to shut up the dissent to it. It's just a pointless nerf that doesn't improve the plastic situation at all and negatively affects a wide range of jobs.

That's why i made a PR to remove this as you can see above.

**Comment by sneksnek** [comment]
> The actual, overarching argument is "this will force chem to make plastic" which is not a problem that happens often and can be solved by asking, ordering a cheap crate from cargo, ordering a machine from cargo that can make infinite plastic from biomass... If you just want more plastic, could have added that recipe to the base biogenerator. Instead doing the wide-scale beaker nerf is not a good solution. Also thanks for confirming that service was just not considered at all - "the people that it matters for".
> 
> > "If I'm frank, I haven't seen many points addressing the change based on it's actual merit. Saying "me and the group of people I know don't like this" is not a strong enough argument on its own. I recommend trying it for a while, trying the new XL beakers, and avoiding having a knee-jerk reaction to a change. I proposed this change with the thought that it wouldn't drastically affect balance or flow since the option for a 120u beaker is still very accessible to the people that it matters for."
> 
> ...a change being disliked by the players IS an absolutely good reason for it to not be there. Telling people to "wait a while" on a PR that was fully merged is just trying to shut up the dissent to it. It's just a pointless nerf that doesn't improve the plastic situation at all and negatively affects a wide range of jobs.
> 
> That's why i made a PR to remove this as you can see above.

I've already addressed these first points on your first PR where you continue to misrepresent my argument and lie.

Players disliking something yet having no concrete argument for why it's a bad change is not a strong enough reason to change it.

Saying to give a reasoned change a try is not silencing opinion. That makes no sense here on this PR or anywhere else.
